app/colorCodeList.js

```diff
@@ -0,0 +1,16 @@
+import {majorColors,minorColors} from './colorCoder.js';
+const colorPairs = [];
+let pairNo = 1;
+for(let i=0;i<majorColors.length;i++){
```

Comment:
Enclose code in function and call it separately. Otherwise a client can print only once, needing to restart everytime it needs to print again.